scalability_and_performance/ztp-factory-install-clusters.adoc

```diff
@@ -0,0 +1,140 @@
+// Telcodocs-403 assembly
+:_content-type: ASSEMBLY
+[id="ztp-factory-workflows"]
+= Zero touch provisioning (ZTP) for factory workflows
+include::_attributes/common-attributes.adoc[]
+:context: ztp-factory-install
+
+:FeatureName: ZTP for factory workflows
+
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+toc::[]
+Zero touch provisioning (ZTP) for factory workflows accelerates the deployment of OpenShift with pre-certified hardware and software for rapid edge deployments.
+
+ZTP for factory workflows enables original equipment manufacturer (OEM) partners to pre-install OpenShift at their factory and build turnkey solutions on their hardware. This approach is well suited to a range of different industries such as healthcare, manufacturing, aerospace, to media, entertainment, retail, and telecommunications.
+
+ZTP for factory workflows installs the components that enable you to use {product-title} as a disconnected hub cluster. This hub cluster is then able to deploy edge clusters that can be shipped off site for final configuration.
+
+At the factory, the OEM partner first deploys a hub {product-title} cluster and then uses the hub cluster to deploy one or more edge clusters at scale.
+
+Different topologies are supported using the ZTP factory workflows. The hub cluster can be a single-node OpenShift cluster deploying multiple single-node OpenShift edge clusters or a compact cluster deploying 3 control plane and 1 worker node edge clusters at scale.
+
+The following cluster topologies are supported:
+
+.Cluster topologies
+[options="header"]
+|====
+|Hub|Edge
+|Compact
+a|3 + 1 (3 control plane nodes and 1 worker node)
+
+Compact (3 control plane nodes also able to act as worker nodes)
+
+Single-node OpenShift (Control plane and worker node on a single node)
+| Single-node OpenShift
+a|3 + 1
+
+Compact
+
+Single-node OpenShift
+|====
+
+Whatever the topology, the hub cluster uses {rh-rhacm-first} and the Assisted Installer (AI) to install edge clusters at scale by using zero touch provisioning (ZTP).
+
+When the edge cluster is installed, it can be shipped to the customer onsite locations and there the end customer unboxes it, and configures the edge cluster making it fully operational.
+
+A high level overview of the zero touch provisioning (ZTP) for factory workflows is as follows:
+
+=== Create the hub cluster
+
+* Install the {product-title} hub cluster in a factory setting.
+* Install OpenShift Data Foundation on the hub cluster.
+* Install the factory pipeline tools and pipeline.
+* Disconnect the hub cluster from the internet.
+* Create bootstrapping IP address pool.
+
+=== Create the edge cluster
+
+* Create the edge cluster.
+* Connect the edge cluster.
+* Start the pipeline.
+* Disconnect the edge cluster.
+
+=== Configure the edge cluster
+
+* Configure the edge cluster on-site.
```

Comment:
Seems redundant. We have a similar line in 2 lines below.

Reply:
removed this section as it does not add anything